src/pages/Account/TransactionRows/BridgeTransactionRow.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  CustomLinkIcon,
+  GridCard,
+  NetworkLink,
+  NetworkName,
+  Status,
+  TokenDetails,
+  TranasctionDetails,
+  TypeDetails,
+} from '../Account.styles'
+import { type BridgeTransaction } from '../Account.types'
+import { TokenIcon } from '../TokenIcon'
+
+interface BridgeTransactionRowProps {
+  transaction: BridgeTransaction
+}
+
+export function BridgeTransactionRow({ transaction }: BridgeTransactionRowProps) {
+  const { type, status, from, to, confirmedTime, logs } = transaction
+
+  const fromNetwork = from?.chainId ? getNetworkInfo(Number(from?.chainId)) : undefined
+  const toNetwork = to?.chainId ? getNetworkInfo(Number(to?.chainId)) : undefined
+  const { bridgeId } = transaction
+
+  const fromLink = getExplorerLink(logs[0]?.chainId, logs[0]?.txHash, 'transaction')
+  const toLink =
+    logs[1]?.chainId !== undefined && logs[1]?.txHash !== undefined
+      ? getExplorerLink(logs[1]?.chainId, logs[1]?.txHash, 'transaction')
+      : undefined
+
+  return (
+    <GridCard status={status.toUpperCase()}>
+      <TokenDetails>
+        <Flex flexDirection="column">
+          <Flex alignItems="center">
+            <TokenIcon symbol={from.token} address={from.tokenAddress} chainId={from.chainId} />
+            <Flex flexDirection="column">
+              <Box>{`${formatNumber(from.value, false, true)}`}</Box>
+              <Box sx={{ fontSize: '0.8em' }}>{from.token}</Box>
+            </Flex>
+          </Flex>
+          <Box sx={{ mt: 1 }}>
+            <NetworkLink href={fromLink} rel="noopener noreferrer" target="_blank">
+              <CustomLinkIcon size={12} />
+              {fromNetwork?.name}
+            </NetworkLink>
+          </Box>
+        </Flex>
+      </TokenDetails>
+
+      <TokenDetails>
+        <Flex flexDirection="column">
+          <Flex alignItems="center">
+            <TokenIcon symbol={to.token} address={to.tokenAddress} chainId={to.chainId} />
+            <Flex flexDirection="column">
+              <Box>{`${formatNumber(to.value, false, true)}`}</Box>
+              <Box sx={{ fontSize: '0.8em' }}>{to.token}</Box>
+            </Flex>
+          </Flex>
+          <Box sx={{ mt: 1 }}>
+            {toLink ? (
+              <NetworkLink href={toLink} rel="noopener noreferrer" target="_blank">
+                <CustomLinkIcon size={12} />
+                {toNetwork?.name}
+              </NetworkLink>
+            ) : (
+              <NetworkName>{toNetwork?.name}</NetworkName>
+            )}
+          </Box>
+        </Flex>
+      </TokenDetails>
+
+      <TranasctionDetails justifyContent="start">
+        <Flex flexDirection="column" alignContent={'center'}>
+          <Box>- -</Box>
+        </Flex>
+      </TranasctionDetails>
+
+      <TypeDetails>
+        <Box color="#8780BF">{type}</Box>
+        <Box fontWeight="600">{bridgeId}</Box>
+      </TypeDetails>
+
+      <TranasctionDetails>
+        <Status status={status.toUpperCase()}>{status}</Status>
```

Comment:
I see we use `.toUpperCase` here but not in `SwapTransactionRow.tsx`. Shouldn't they do the same thing?

Reply:
The swap status was already uppercase. For consistency, I can do that here as well.